### PR TITLE
chore: add feature flag for Next plugin

### PR DESCRIPTION
**Pull request by ascorbic**
Adds a feature flag for the latest version of the Next plugin

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

**Comment by ascorbic** [comment]
@erezrokah I'm not sure if the failing test is correct. _Does_ it need to have a different major in each entry?

**Comment by erezrokah** [comment]
> @erezrokah I'm not sure if the failing test is correct. _Does_ it need to have a different major in each entry?

This seems like a bug in the test. I think it should not enforce it when adding a feature flagged version

File: site/plugins.json [hunk 1]
```diff
@@ -479,6 +479,10 @@
     "repo": "https://github.com/netlify/netlify-plugin-nextjs",
     "version": "4.2.8",
     "compatibility": [
+      {
+        "version": "4.3.1",
+        "featureFlag": "build_plugins_use_prerelease"
```
Comment (erezrokah) [inline site/plugins.json:484]:
```suggestion
        "featureFlag": "build_plugins_next_js_use_prerelease"
```

**Review by erezrokah — CHANGES_REQUESTED** [review]

File: site/plugins.json (hunk 1 above)
Comment (ascorbic) [reply site/plugins.json:484]: I was thinking it could be used for any prerelease plugin

Comment (erezrokah) [reply site/plugins.json:484]: Ah, but what if you want to enable it only for a single plugin?

**Review by erezrokah — DISMISSED** [review]

**Review by erezrokah — APPROVED** [review]

**Pull request merged**